docs/HowToFAQ/FAQGhostSideEffects.md

```diff
@@ -0,0 +1,32 @@
+---
+title: How can ghost code call methods with side effects?
+---
+
+## Question
+
+How can ghost code call methods with side effects?
+
+## Answer
+
+Ghost code may not have any effect on non-ghost code, but ghost code can have effects on ghost code.
```

Comment:
Does it make sense to say "non-ghost state" or "ghost state", instead? Ghost code can only affect other ghost code through ghost state, so it's probably correct either way, but I feel like "state" is more direct.